readme.md

```diff
@@ -46,7 +47,8 @@ From sources:
 pip3 install python-telegram
 git clone git@github.com:paul-nameless/tg.git
 cd tg
-PYTHONPATH=. python3 tg/main.py
+PYTHONPATH=. 
+python3 tg/main.py
```

Comment:
so, need command should be `python  -m tg`, isn't it? then `PYTHONPATH` could be omitted.

Reply:
Now that I think of it... You're using `flit` as a package manager, so the readme should reflect that...

Reply:
agree, it is worth to add this to README. it will make life easier for people who will be adding brew package and others